nmdc_schema/migrators/migrator_from_X_to_PR23.py

```diff
@@ -0,0 +1,57 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+import difflib
+
+class Migrator_from_X_to_PR23(MigratorBase):
+    """Migrates data from schema X to PR23"""
+
+    def __init__(self, *args, **kwargs) -> None:
+        """Invokes parent constructor and populates collection-to-transformations map."""
+
+        super().__init__(*args, **kwargs)
+
+        # Populate the "collection-to-transformers" map for this specific migration.
+        self.agenda = dict(
+            metagenome_assembly_set=[self.execution_resource_update], metagenome_annotation_activity_set=[self.execution_resource_update],
+            metatranscriptome_activity_set=[self.execution_resource_update], mags_activity_set=[self.execution_resource_update],
+            metagenome_sequencing_activity_set=[self.execution_resource_update], read_qc_analysis_activity_set=[self.execution_resource_update],
+            read_based_taxonomy_analysis_activity_set=[self.execution_resource_update], metabolomics_analysis_activity_set=[self.execution_resource_update],
+            metaproteomics_analysis_activity_set=[self.execution_resource_update], nom_analysis_activity_set=[self.execution_resource_update]
+        )
```

Comment:
I'd prefer each key of this dictionary be on its own line; i.e.

```diff
        self.agenda = dict(
-            metagenome_assembly_set=[self.execution_resource_update], metagenome_annotation_activity_set=[self.execution_resource_update],
-            metatranscriptome_activity_set=[self.execution_resource_update], mags_activity_set=[self.execution_resource_update],
-            metagenome_sequencing_activity_set=[self.execution_resource_update], read_qc_analysis_activity_set=[self.execution_resource_update],
-            read_based_taxonomy_analysis_activity_set=[self.execution_resource_update], metabolomics_analysis_activity_set=[self.execution_resource_update],
-            metaproteomics_analysis_activity_set=[self.execution_resource_update], nom_analysis_activity_set=[self.execution_resource_update]
+            metagenome_assembly_set=[self.execution_resource_update],
+            metagenome_annotation_activity_set=[self.execution_resource_update],
+            metatranscriptome_activity_set=[self.execution_resource_update],
+            mags_activity_set=[self.execution_resource_update],
+            metagenome_sequencing_activity_set=[self.execution_resource_update],
+            read_qc_analysis_activity_set=[self.execution_resource_update],
+            read_based_taxonomy_analysis_activity_set=[self.execution_resource_update],
+            metabolomics_analysis_activity_set=[self.execution_resource_update],
+            metaproteomics_analysis_activity_set=[self.execution_resource_update],
+            nom_analysis_activity_set=[self.execution_resource_update]
        )
```

I find it easier to see the list of collections (and the lengths of the "transformation pipelines") at a glance that way.

Reply:
Updated! Thanks!